src/config/dark-sites.config

```diff
@@ -270,13 +279,15 @@ kiwifarms.net
 knockout.chat
 kodenames.io
 krunker.io
+lab.openbloc.fr/droneWorld/
```

Comment:
![](https://i.imgur.com/xlvwAsB.png) White.

Reply:
Brave blocks something, has white flash, will remove it.